dockerfiles/Dockerfile.shellcheck

```diff
@@ -1,5 +1,9 @@
-FROM    koalaman/shellcheck-alpine:v0.7.1
-RUN     apk add --no-cache bash make
+# syntax=docker/dockerfile:1.3
+
+FROM koalaman/shellcheck-alpine:v0.7.1 AS shellcheck
 WORKDIR /go/src/github.com/docker/cli
-ENV     DISABLE_WARN_OUTSIDE_CONTAINER=1
-COPY    . .
+RUN apk add --no-cache bash
```

Comment:
I think the only reason we installed `bash` may have been because either the script used bash, or because of the `set -eo pipefail` (which is non-posix), so perhaps we don't even need this